src/components/Modal/Modal.module.scss

```diff
@@ -0,0 +1,20 @@
+.ModalWrapper {
+  display: none;
+
+  &Active {
+    align-items: center;
+    background: var(--GraySecondary--translucent);
+    display: flex;
+    height: 100vh;
+    justify-content: center;
+    left: 0;
+    position: fixed;
+    top: 0;
+    width: 100vw;
```

Comment:
Any reason not to put the height and width closer together here?

Reply:
The properties are listed in alphabetical order. There's really no reason for this other than it being a habit I picked up years ago...

Reply:
Oh nice never seen that. I can see how that is useful especially with really long styles

Reply:
that's fine...it's just a nitpick / non-blocker